### PR TITLE
Add CVE Whitelist test cases into nightly test

### PR DESCRIPTION
**Pull request by danfengliu**
Add CVE Whitelist test cases into nightly test, first test case is system CVE Whitelist test.

Signed-off-by: danfengliu <danfengl@vmware.com>

### CODE REVIEW

File: tests/resources/Harbor-Pages/Configuration.robot [hunk 1]
```diff
@@ -235,63 +235,75 @@ Disable Read Only
...
 View GC Details
-    Click Element  xpath=${gc_log_details_xpath}
+    Retry Element Click  xpath=${gc_log_details_xpath}
     Sleep  2
 
 Switch To GC History
     Retry Element Click  xpath=${gc_log_xpath}
     Retry Wait Until Page Contains  Job
 
+Create A New System CVE Whitelist
```
Comment (reasonerjt) [inline tests/resources/Harbor-Pages/Configuration.robot]: This is "Add Items To System CVE Whitelist"

File: tests/resources/Harbor-Pages/Project-Config.robot [hunk 2]
```diff
@@ -54,3 +54,11 @@ Prevent Running Should Be Selected
 Auto Scan Should Be Selected
     Checkbox Should Be Selected  //hbr-project-policy-config//input[@name='scan-image-on-push']
 
+Create A New Project CVE Whitelist
```
Comment (reasonerjt) [inline tests/resources/Harbor-Pages/Project-Config.robot]: Add Items to Project CVE Whitelist

File: tests/robot-cases/Group1-Nightly/Clair.robot [hunk 3]
```diff
@@ -143,3 +143,56 @@ Test Case - Project Level Image Serverity Policy
...
+    Init Chrome Driver
+    ${d}=    Get Current Date    result_format=%m%S
+    ${image}=    Set Variable    redis
+    ${sha256}=  Set Variable  9755880356c4ced4ff7745bafe620f0b63dd17747caedba72504ef7bac882089
+    ${signin_user}=    Set Variable  user025
+    ${signin_pwd}=    Set Variable  Test1@34
+    Sign In Harbor    ${HARBOR_URL}    ${signin_user}    ${signin_pwd}
+    Create An New Project    project${d}
+    Push Image    ${ip}    ${signin_user}    ${signin_pwd}    project${d}    ${image}    sha256=${sha256}  tag=${sha256}
+    Pull Image    ${ip}    ${signin_user}    ${signin_pwd}    project${d}    ${image}    tag=${sha256}
```
Comment (reasonerjt) [inline tests/robot-cases/Group1-Nightly/Clair.robot]: why pull it after pushing?

Comment (danfengliu) [reply tests/robot-cases/Group1-Nightly/Clair.robot]: Just to make sure image can be pulled before any configurations.

File: tests/resources/Harbor-Pages/Project-Config.robot (hunk 2 above)
Comment (danfengliu) [reply tests/resources/Harbor-Pages/Project-Config.robot]: Yes, I will modify it.

File: tests/resources/Harbor-Pages/Configuration.robot (hunk 1 above)
Comment (danfengliu) [reply tests/resources/Harbor-Pages/Configuration.robot]: Yes, I will modify it.

**Pull request merged**